cairo/tests/utils/args_gen.py

```diff
@@ -647,6 +647,19 @@ def to_transient_storage(self) -> TransientStorage:
         return ts
 
 
+builtins_exception_classes = inspect.getmembers(
+    sys.modules["builtins"],
+    lambda x: inspect.isclass(x) and issubclass(x, Exception),
+)
+
+builtins_exception_mappings = {
+    (
+        "builtins",
+        f"{name}",
+    ): cls
+    for name, cls in builtins_exception_classes
+}
```

Comment:
```suggestion
// TODO: builtins exceptions are defined in the ethereum.exceptions module for now.
// Refactor if required
builtins_exception_mappings = {
    (
        "ethereum",
        "exceptions",
        f"{name}",
    ): cls
    for name, cls in builtins_exception_classes
}
```

no impact yet, however i dont want to risk surprises later, and dont think its important enough to make a new package specifically for builtin errors yet